wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -799,7 +799,7 @@ private Subject sessionUser() {
 
     private String sessionUserNameOr(String nameIfNoUser) {
         final Subject subject;
-        if (EnvironmentContext.getCurrent() != null && (subject = EnvironmentContext.getCurrent().getSubject()) != null) {
+        if ((subject = EnvironmentContext.getCurrent().getSubject()) != Subject.ANONYMOUS) {
```

Comment:
@sleshchenko maybe a test is missing in https://github.com/sleshchenko/che/blob/cd0fb094cfac053c8bb618fac13200ac1880f17b/wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/WorkspaceManagerTest.java
to ensure it will always work as designed